README.md

```diff
@@ -74,10 +74,28 @@ proc waitFor*[T](fut: Future[T]): T =
   return fut.read()
 ```
 
+### Async procedures and methods
+
+The `{.async.}` pragma will transform a procedure (or a method) returning a
+specialised `Future` type into a closure iterator. If there is no return type
```

Comment:
`{.async.}` pragma do not transform procedure into closure iterator. It transforms procedure to 3 procedures. One of this 3 procedures is a closure iterator.

Reply:
We don't want to scare them off so soon. Let's break it to them in small chunks.

Reply:
If you want to document `poll()` document everything properly. `poll()` is exactly not for newcomers.